org.eclipse.winery.yaml.common/src/main/java/org/eclipse/winery/yaml/common/validator/ImportVisitor.java

```diff
@@ -32,6 +33,8 @@
 import org.eclipse.winery.yaml.common.validator.support.Parameter;
 import org.eclipse.winery.yaml.common.validator.support.Result;
 
+import com.google.common.collect.Sets;
```

Comment:
Is it possible to use another library or join the sets in other ways? The process to get a new library in is hard and for this short usage, it seems, the equivalent code is easy to write.

If you find a similar method at https://www.eclipse.org/collections/, it is OK to use that, because Eclipse Collections is part of the Eclipse universe.